test/common_nn.py

```diff
@@ -854,6 +856,22 @@ def test_cuda(self, test_case):
                 for cpu_d_p, gpu_d_p in zip(cpu_param[1], gpu_param[1]):
                     test_case.assertEqual(cpu_d_p, gpu_d_p, 2e-4)
 
+            # Run double-backwards on CPU and GPU and compare results
+            # TODO: BatchNorm double-backwards is currently broken, see
+            # #4422; remove this when it is fixed.
+            if self.check_gradgrad and not self.get_name().startswith('test_BatchNorm'):
```

Comment:
can we explicitly write check_gradgrad=False in all the test specifications where we need it?  This kind of code leads to people realizing months later that we never turned on the tests.

Reply:
So... would you like me to do this "correctly" or "easily"? It's not correct to put `check_gradgrad=False` in the batchnorm tests because that also disables the CPU double backward tests and those are passing on HEAD. So the correct thing is to add another variable to control this but then I decided it was too much work and added a special case (which I promise to remove...)

Reply:
Sure, add a different flag :).

Reply:
`FIXME_no_gradgrad=True`

Reply:
Done.